js/web/src/components/NavDrawer.js

```diff
@@ -132,9 +131,7 @@ const NavDrawer = () => {
       {
         <Box key={'left'}>
           <StyledMenuButton onClick={toggleDrawer(true)}>
-            <Icon>
-              <img src={hamburger} height={25} width={25} />
-            </Icon>
+              <Menu />
```

Comment:
i think we should we stick w the icon from spevacks design. The spacing in it follows some general theme spacing